analysers/experiments/cpu.py

```diff
@@ -15,6 +15,16 @@
 from pyspark_cassandra import RowFormat
 from pyspark import SparkConf
 
+def getHostCores(sc, cassandraKeyspace, hostID):
```

Comment:
Why do you need this function? Why don't you refer to the container properties where you have (if less than the maximum) the number of cores? You should refer to the host only if you don't have the number of core information in the container properties.